### DISCUSSION TITLE
Problems with events shorter than 1 day in duration

### DISCUSSION THREAD

**Discussion by lcabrera** [csgoh/roadmapper#33] [Announcements]
Currently, the minimum duration of an item is 1 day. If the duration is less, an exception is generated.

Using 1 day as the minimum unit in this type of diagrams, it is impossible to represent events with a lower temporal resolution.

Would it be possible to prepare this project to be able to represent temporal units smaller than 1 day?

Thanks in advance...

**Comment by csgoh** [comment]
This is a feature request not a issue or defect. Moving this to Discussion page.

**Comment by csgoh** [comment]
I understand your use case, but Roadmapper was intended to be used as a tool to generate high-level roadmap and not project gantt chart.

**Comment by lcabrera** [comment]
Ok, I understand, it's normal.

However, maybe this option enables a new type of 'use cases' not contemplated so far.

In any case, thanks for the development and for the effort.

Thx
